cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1553,6 +1553,12 @@
   </property>
 
   <property>
+    <name>metadata.kafka.broker.list</name>
+    <value>127.0.0.1:${kafka.bind.port}</value>
+    <description>Kafka broker list to which metadata update notifications are published</description>
+  </property>
+
+  <property>
     <name>metadata.updates.kafka.topic</name>
     <value>metadata-updates</value>
```

Comment:
Since we are publishing to external kafka now, it would be better to prefix the topic name with `cdap-`.